internal/provider/provider.go

```diff
@@ -135,10 +135,7 @@ func (p *hookdeckProvider) Configure(ctx context.Context, req provider.Configure
 	tflog.Debug(ctx, apiBase+" "+apiKey)
 
 	// Create a new Hookdeck client using the configuration values
-	client := hookdeckClient.NewClient(
-		hookdeckClient.WithBaseURL(apiBase),
-		hookdeckClient.WithAuthToken(apiKey),
-	)
+	client := sdkclient.InitHookdeckSDKClient(apiBase, apiKey, p.version)
```

Comment:
@alexluong so this will be dynamic based on the release?

Reply:
Yes, that's "p.version". The value is coming from Terraform depending on which provider version the user is using.

Reply:
👍 https://github.com/hookdeck/terraform-provider-hookdeck/blob/062102e5608b0d7c17c288a4f897cfbf0300a606/internal/provider/provider.go#L29